css/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+    margin: 0px;
+    padding: 0px;
+    min-width: 1200px
+}
+.wrapper {
+    width: 1200px;
+    margin: auto;
+}
+.block {
+    line-height: 150px;
+    height: 150px;
+}
+header .block {
+    width: 400px;
+}
+.block:first-child {
```

Comment:
are you sure in this selector? it will apply `float:left` for each `.block` under parent tag
in you example: header and footer